Test/git-issues/git-issue-1094.dfy

```diff
@@ -98,11 +98,10 @@ datatype MStatus = Okay | Error(code: int) {
   predicate method IsFailure() {
     Error?
   }
-  method PropagateFailure() returns (m: MStatus)
+  function method PropagateFailure(): MStatus
```

Comment:
Please make this Dafny 4 ready. Perhaps change the syntax and putting an option in the RUN command